### PR TITLE
Update RTL14 for 0.9

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: Spec/RealtimeClientChannel.swift [hunk 1]
```diff
@@ -2325,6 +2325,48 @@ class RealtimeClientChannel: QuickSpec {
...
+                    }
+
+                    waitUntil(timeout: testTimeout) { done in
+                        let errorProtocolMessage = AblyTests.newErrorProtocolMessage()
+                        errorProtocolMessage.action = .Error
+                        errorProtocolMessage.channel = "foo"
+
+                        let partialDone = AblyTests.splitDone(2, done: done)
+
+                        channel.once(.Error) { stateChange in
```
Comment (tcard) [inline Spec/RealtimeClientChannel.swift]: `.Error` doesn't exist anymore. Only FAILED should be emitted as per RTL14.

**Comment by tcard** [comment]
Why was this merged?

**Comment by tcard** [comment]
Ah OK, because of https://github.com/ably/ably-ios/pull/544#issuecomment-267021928.

**Comment by ricardopereira** [comment]
@tcard Because I removed the `.Error` event and I thought it would be the only thing to change.

**Comment by tcard** [comment]
Ah right, sorry, I was misreading my own change request...

**Pull request merged**